src/test/java/com/thebois/models/world/structures/TownHallTests.java

```diff
@@ -0,0 +1,53 @@
+package com.thebois.models.world.structures;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TownHallTests {
+
+    @Test
+    public void TownHallIsAlwaysCompleted() {
+        // Arrange
+        final IStructure house = StructureFactory.createStructure(
+            StructureType.TOWN_HALL,
+            new Position());
+
+        // Act
+        final boolean isComplete = house.isCompleted();
+
+        // Assert
+        assertThat(isComplete).isTrue();
+    }
+
+    @Test
+    public void TownHallHasAlways1AsBuiltRatio() {
+        // Arrange
+        final IStructure house = StructureFactory.createStructure(
+            StructureType.TOWN_HALL,
+            new Position());
+
+        // Act
+        final float builtRatio = house.getBuiltRatio();
+
+        // Assert
+        assertThat(Float.compare(builtRatio, 1) >= 0).isTrue();
```

Comment:
Question
Is this how we want to write these kinds of tests?
Because when I've been writing these I've just done this:
`assertThat(actualValue).isEqualTo(expectedValue)`

Probably doesn't matter, was just wondering.

Reply:
it should be `isGreaterThanOrEqualTo`, It is now fixed, thank 👍